server/http_config_device_handler.go

```diff
@@ -111,21 +126,6 @@ func deviceConfigMap[T any](class templates.Class, dev config.Device[T]) (map[st
 		}
 	}
 
-	if dc["config"] == nil {
```

Comment:
Vorschlag: nicht nicht auf die config checken sondern den Titel aus Others nur ergänzen wenn `dc.config.title` noch keinen Wert hat.

Reply:
Is ja nu schon: https://github.com/evcc-io/evcc/pull/21956/commits/34256e906be14d7f69c600a7735f67acae32a88d. Tut trotzdem nicht. Mhhm.

Reply:
Ich vermute Du gehst auf deviceTitle, nicht title? Das würde es dann erklären: https://github.com/evcc-io/evcc/pull/21956/commits/7b202156abc7a4106ec033e62a5acec0353d2ea5